source/FunkinLua.hx

```diff
@@ -1377,6 +1377,16 @@ class FunkinLua {
 			leSprite.antialiasing = ClientPrefs.globalAntialiasing;
 			PlayState.instance.modchartSprites.set(tag, leSprite);
 		});
+		
+		// Lua_helper.add_callback(lua, "makeJsonI8LuaSprite", function(tag:String, image:String, x:Float, y:Float, ?spriteType:String = "sparrow") {
```

Comment:
I recommend shorting it to `makeI8LuaSprite` or `makeI8Sprite`- idrk know at this point.

Reply:
> I recommend shorting it to `makeI8LuaSprite` or `makeI8Sprite`- idrk know at this point.

thats unused code lmao, it isnt even applied its commented

Reply:
> > I recommend shorting it to `makeI8LuaSprite` or `makeI8Sprite`- idrk know at this point.
> 
> thats unused code lmao, it isnt even applied its commented

I didn't even notice XD